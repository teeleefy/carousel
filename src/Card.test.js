import React from "react";
import { render, fireEvent} from "@testing-library/react";
import Card from "./Card";
import image1 from "./image1.jpg";

it("renders without crashing", function() {
  let { debug } = render(<Card 
    caption="This is a photo."
    src={image1}
    currNum="1"
    totalNum="3"/>);
    debug();
});

it("matches snapshot", function() {
  const { asFragment } = render(<Card caption="This is a photo."
  src={image1}
  currNum="1"
  totalNum="3"/>);
  expect(asFragment()).toMatchSnapshot();
});
