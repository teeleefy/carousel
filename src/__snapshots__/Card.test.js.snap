// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      This is a photo.
    </h4>
    <img
      alt="This is a photo."
      class="Card-image"
      src="image1.jpg"
    />
    <small
      class="Card-small"
    >
      Image 1 of 3.
    </small>
  </div>
</DocumentFragment>
`;
